### DISCUSSION TITLE
記事を書く画面のスクロールバーのスライダ色の視認性が悪い

### DISCUSSION THREAD

**Discussion by radian-jp** [increments/qiita-discussions#895] [要望・アイデア・不具合]
タイトル通りですが、白すぎて殆ど見えません。過去記事編集のために久々に触ったのですが、以前からこんなに見づらかったですかね…？
好みの問題かもしれませんが、エディタ領域のテキストのフォントもなんだかぼやけたようなフォントで見づらくなった気がします。プレビュー領域のフォントは読みやすいのですが。

![記事を書く画面](https://github.com/user-attachments/assets/29113156-c185-4c16-b00e-d85958725189)

ホーム画面のスクロールバーは普通に見えるので、なんだか一貫性がない気がします。

![ホーム画面](https://github.com/user-attachments/assets/ed42da76-388a-40f6-9bbe-595fcbdfb153)

**Comment by kabechiyo** [comment]
@radian-jp 
ご報告ありがとうございます🙇

スクロールバーのスライダ色についてコントラスト比が低くなってしまっているため、修正いたします！
また、フォントが読みづらい点についても修正できないか調査し、改善を検討します！

**Reply by kabechiyo** [reply]
@radian-jp 
お待たせしました！
スクロールバーのスライダ色のコントラスト比が高くなるように修正いたしました。
https://qiita.com/release-notes/2024-08-29

この度はご報告いただきありがとうございます🙏
フォントが読みづらい点についても追って調査・改善検討いたします！

**Reply by radian-jp** [reply]
スクロールバーが改善されているのを確認しました。
対応ありがとうございます！

**Reply by kabechiyo** [reply]
@radian-jp 
お待たせいたしました。

エディタのフォントがぼやけてしまう問題についても修正を行いました！
エディタに和文フォントの指定がないことが原因だったため、指定を追加いたしました。
https://qiita.com/release-notes/2024-10-10

この度はご報告いただきありがとうございました 🙏

**Reply by radian-jp** [reply]
![screenshot_after](https://github.com/user-attachments/assets/a842a01c-7ddf-4008-879c-6a3c02965dbc)

フォントの方も対応ありがとうございます！
前回アップしたスクリーンショットのフォントがぼやけていた部分を撮り直してみましたが、
確かにぼやけが解消されているようです。

**Reply by tanaka-asami415** [reply]
@radian-jp 
ご確認いただきありがとうございます！
解消されたとのこと、何よりです🙌✨
